.storybook/polaris-readme-loader.js

```diff
@@ -187,52 +140,54 @@ function parseCodeExamples(data, file) {
     .split('###');
 
   const [, ...allExamples] = examplesAndHeader;
-  const filePath = file.split('polaris-react/').slice(-1)[0];
 
   if (allExamples.length === 0) {
-    throw new Error(
-      chalk`🚨 {red [${matter.data.name}]} No examples found in ${filePath}`,
+    console.log(
+      chalk`🚨 {red [${
+        matter.data.name
+      }]} No examples found. For troubleshooting advice see https://github.com/Shopify/polaris-react/blob/master/documentation/Component%20READMEs.md#troubleshooting'`,
```

Comment:
There's an extraneous `'` at the end of the URL:

```suggestion
      }]} No examples found. For troubleshooting advice see https://github.com/Shopify/polaris-react/blob/master/documentation/Component%20READMEs.md#troubleshooting`,
```

Reply:
I am blind, thanks